src/main/scala/com/fulcrumgenomics/bam/api/SamRecord.scala

```diff
@@ -289,6 +300,9 @@ object SamRecord {
   /** The maximum insert size that can be stored in a [[SamRecord]]. */
   val MaximumInsertSize: Int = SAMRecord.MAX_INSERT_SIZE
 
+  /** The SAM tag for the mate's cigar. */
+  val McTag: String = MC.toString
```

Comment:
This is a slippery slope.  I we _want_ to include them, I'd create `object SamTags` to store them in.  Honestly, I don't see the full utility of having such a facade and not just using either "MC" hardcoded as we've done, or `htsjdk.samtools.SAMTag.MC` directly

Reply:
I don't think any performance boost is meaningful so I typically do it because I'm allergic to string literals. I can rip it out!